src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskClaimByIdControllerTest.java

```diff
@@ -102,6 +105,51 @@ public void user_should_claim_task_when_role_assignment_verification_passed() {
         common.cleanUpTask(taskId);
     }
 
+    @Test
+    public void user_should_claim_task_and_claim_action_recorded_in_replicate_db() throws InterruptedException {
+
+        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("processApplication",
+                                                                       "Process Application");
+        initiateTask(taskVariables);
+
+        common.setupWAOrganisationalRoleAssignment(caseworkerCredentials.getHeaders(), "tribunal-caseworker");
+
+        TimeUnit.SECONDS.sleep(10);
+
+        String taskId = taskVariables.getTaskId();
+        Response result = restApiActions.post(
+            ENDPOINT_BEING_TESTED,
+            taskId,
+            caseworkerCredentials.getHeaders()
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.NO_CONTENT.value());
+
+        Response resultHistory = restApiActions.get(
+            "/task/{task-id}/history",
+            taskId,
+            caseworkerCredentials.getHeaders()
+        );
+
+        resultHistory.prettyPrint();
+        resultHistory.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .body("task_history_list.size()", equalTo(2))
+            .body("task_history_list.get(0).state", equalTo("UNASSIGNED"))
+            .body("task_history_list.get(0).assignee", equalTo(null))
+            .body("task_history_list.get(0).updated_by", notNullValue())
+            .body("task_history_list.get(0).updated", notNullValue())
+            .body("task_history_list.get(0).update_action", equalTo("Configure"))
+            .body("task_history_list.get(1).state", equalTo("ASSIGNED"))
+            .body("task_history_list.get(1).assignee", notNullValue())
+            .body("task_history_list.get(1).updated_by", notNullValue())
+            .body("task_history_list.get(1).updated", notNullValue())
+            .body("task_history_list.get(1).update_action", equalTo("Claim"));
+
+        common.cleanUpTask(taskId);
```

Comment:
Now we know the replication for insert and update works we should probably cover one for delete.  Could be another ticket but might be much quicker to add here?

Reply:
Test added for Terminated task.